custom_components/octopus_intelligent/octopus_intelligent_system.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, hass, *, api_key, account_id, off_peak_start, off_peak_end):
             # Name of the data. For logging purposes.
             name="Octopus Intelligent",
             # Polling interval. Will only be polled if there are subscribers.
-            update_interval=timedelta(seconds=300),
+            update_interval=timedelta(seconds=120),
```

Comment:
Thanks for the PR. Much appreciated.

I'm a bit reluctant to increase the amount of requests we make to octopus. Given the recent noise on this and the fact most were happy with 300s before, I'm going to revert this change but keep the timeout increase.

Reply:
I looked into that and they allow 10 requests a min, so well within that with the 2 min poll; the issue is their sometimes long responses. Maybe they have locking issues, not sure. 2 mins is working fine for me.

Reply:
It is true that some users jump to conclusions and incorrectly assume that any API failures are a result of rate limiting. On the other hand, response times of 30+ seconds may be a sign of server overload. I note that the proposed increased timeout of 60 seconds is 50% of the proposed reduced polling interval of 120 seconds. Think about it: If the servers took 60 seconds to reply to a request, hitting it with another request just 60 seconds later “feels a bit mean.” (Although Octopus should get their act together on server capacity and introduce real bitting rate limits if that’s an issue for them.)

My 2 cents would be for this _“Unavailable sensor fix”_  PR to focus on increasing the timeout only, which is what fixes the issue. The polling interval in my opinion should be configurable instead of hardcoded (hass config flow).